src/components/legend/draw.js

```diff
@@ -356,12 +311,12 @@ module.exports = function draw(gd) {
     }
 };
 
-function drawTexts(context, gd, d, i, traces) {
+function drawTexts(gd, context, legendItem) {
```

Comment:
Nice.Things like this make it much easier to read so I'm always happy to see these changes.